### DISCUSSION TITLE
Automatización/Actualización de EPG.

### DISCUSSION THREAD

**Discussion by pepelecho** [K-vanc/Tempest-EPG-Generator#133] [General]
El script es muy completo, salvo que tiene un gran fallo: No hay una manera de actualizar automaticamente la programación EPG. 
Sería conveniente que existiera alguna manera de que la programación se actualizara sola, como, por ejemplo, que la guia xml se mostrara en un script PHP y se actualizara una vez al día. O que pudieses acceder a un script PHP a parte que actualizara la guia. De la forma que está ahora, no es práctico tener que estar entrando continuamente a actualizar la programación.

**Comment by K-vanc** [comment]
Hi,

Sorry i dont know Spanish so had to use translator. As far as I understood(please correct me if I am wrong), you are telling Epg automation is missing and this is a problem. 

Tempest can be run also with commandline(cli) parameters and this is the way to automate epg generation with OS supported schedulers which is Scheduled Tasks(on Windows) and Cron/Crontab (on Unix distros).

Basic command for epg generations is;

php /your/path/tempest.php --epg config=your_config_name

You can set this line directly into cron schedulers or put inside a file (.bat for Windows and .sh for Unix) and set this file to run periodically via schedulers. I also recommend to enable Fusion with level 0 settings so Tempest will check daily grabbed shows, update and preserve already grabbed shows in case of a failure in new run.

I hope this helps

**Comment by K-vanc** [comment]
Hi, closing the discussion since no answer from you
